analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -108,7 +109,10 @@ public void handleMessage(Message msg) {
   @Private static final String OPT_OUT_PREFERENCE_KEY = "opt-out";
   static final String WRITE_KEY_RESOURCE_IDENTIFIER = "analytics_write_key";
   static final List<String> INSTANCES = new ArrayList<>(1);
+  /* This is intentional, and optional. */
```

Comment:
Unclear to me why we can suppress this

Reply:
We're only using the application context. It can't really be leaked as there's always only one. The warning is for catching leaks like activities or views.

Reply:
Added note about application context